src/calibre/ebooks/conversion/plugins/pdb_output.py

```diff
@@ -14,13 +14,13 @@ class PDBOutput(OutputFormatPlugin):
     author = 'John Schember'
     file_type = 'pdb'
     commit_name = 'pdb_output'
-    ui_data = {'formats': tuple(ALL_FORMAT_WRITERS)}
+    ui_data = {'formats': ALL_FORMAT_WRITERS}
```

Comment:
This is converted into a tuple for a reason, sets are not JSON serializable

Reply:
Although looking further down your PR I see you changed ALL_FORMAT_WRITERS to a tuple